tests/unit/type/stream-cgroups.tcl

```diff
@@ -43,6 +43,114 @@ start_server {
         r XGROUP CREATE mystream mygroup $ MKSTREAM
     } {OK}
 
+    test {XREADGROUP parameter validation setup} {
+        r DEL mystream
+        r XADD mystream * field value
+        r XGROUP CREATE mystream mygroup $
+    }
```

Comment:
we should try to make each test independent so that we can use `--only` to run a special test.
but now do other tests depend on this test?